### PR TITLE
fix #38347 zoobeeg.com

### LINKED ISSUE #38347: zoobeeg.com
### Issue URL (Anti Adblock Script)
[NSFW][https://zoobeeg.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fzoobeeg.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.0
Browser: | Chrome
Filters: | AdGuard Base,<br>AdGuard Annoyances

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/38347

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -454,6 +454,19 @@ shotfly.in#%#AG_defineProperty('app_vars.force_disable_adblock', {value: "0"});
 safemaru.blogspot.com###unblocker
 ! https://github.com/AdguardTeam/AdguardFilters/issues/32766
 videogreen.gq#@#.afs_ads
+! https://github.com/AdguardTeam/AdguardFilters/issues/38347
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt]: why not just use CSS rule and exclusions like for FAB script?

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -455,7 +455,7 @@ safemaru.blogspot.com###unblocker
 ! https://github.com/AdguardTeam/AdguardFilters/issues/32766
 videogreen.gq#@#.afs_ads
 ! https://github.com/AdguardTeam/AdguardFilters/issues/38347
-zoobeeg.com#%#AG_abortOnPropertyRead('adBlockDetected');
+zoobeeg.com#$#.pub_300x250.pub_300x250m.pub_728x90.text-ad.textAd.text_ad.text_ads.text-ads.text-ad-links { display: block !important; }
```
Comment (BlazDT) [inline EnglishFilter/sections/antiadblock.txt]: We have global CSS rule, please add domain to it.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 2 above)
Comment (Alex-302) [reply EnglishFilter/sections/antiadblock.txt]: all these rules can be replaced by `zoobeeg.com##.swal2-container`. Easier - better.

Comment (BlazDT) [reply EnglishFilter/sections/antiadblock.txt]: I remember we had problems in the past when we hid swal2-container (some Turkish sites). That modal is sometimes used for login/register etc.

Comment (Alex-302) [reply EnglishFilter/sections/antiadblock.txt]: this site has players only.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**